### PR TITLE
Add docs for user management APIs

### PR DESCRIPTION
**Pull request by aashikam**
## Purpose
$title

### CODE REVIEW

File: en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md [hunk 1]
```diff
@@ -918,6 +918,126 @@ To use the Google Ads connector, first create the connection with your configura
     }
     ```
 
+??? note "uploadUserData"
+    The `uploadUserData` operation uploads the given user data..
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md:922]:
```suggestion
    The `uploadUserData` operation uploads the given user data.
```

**Review by arunans23 — APPROVED** [review]

File: en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md [hunk 2]
```diff
@@ -918,6 +918,126 @@ To use the Google Ads connector, first create the connection with your configura
...
+            <th>Required</th>
+        </tr>
+        <tr>
+            <td><code>customerId</code></td>
+            <td>Required. The ID of the customer whose user lists are being modified.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>operations</code></td>
+            <td>Required. The list of operations to perform on individual user lists. Type: object (UserListOperation)</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md:996]: Shall we link the object from the Google Ads API doc here to maintain consistency?

File: en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md [hunk 3]
```diff
@@ -918,6 +918,126 @@ To use the Google Ads connector, first create the connection with your configura
...
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>partialFailure</code></td>
+            <td>If true, successful operations will be carried out and invalid operations will return errors. If false, all operations will be carried out in one transaction if and only if they are all valid. Default is false.</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md:1001]:
```suggestion
            <td>If true, successful operations will be carried out and invalid operations will return errors. If false, all operations will be carried out in one transaction if and only if they are all valid. The default value is <code>false</code>.</td>
```

File: en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md (hunk 2 above)
Comment (aashikam) [reply en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md]: ec1816096eb3119e728cf4e4b0db4edde3a08ac2

**Review by DinithiDiaz — APPROVED** [review]

**Pull request merged**